examples/extract_mnist_gpu.py

```diff
@@ -11,10 +11,19 @@
 from privacyraven.models.victim import train_four_layer_mnist_victim
 from privacyraven.models.four_layer import FourLayerClassifier
 
-# Create a query function for a target PyTorch Lightning model
-model = train_four_layer_mnist_victim(gpus=1)
 
+# PrivacyRaven's builtin victim functions allow the user to specify the number of GPUs
+# to use during computation via the `gpus` keyword argument.  By default, this is 1, but 
+# setting gpus=torch.cuda.device_count() will let PrivacyRaven / Pytorch use all available GPUs.
```

Comment:
I think putting this comment here is too much information for one of the starter examples.

Reply:
Do you want me to move this comment or just condense it?